tests/utils/test_ensemble.py

```diff
@@ -340,7 +365,7 @@ def test_Ensemble_fit_generator():
         ens = Ensemble(keras_model, n_estimators=2, n_predictions=1)
 
         ens.compile(
-            optimizer=Adam(), loss=categorical_crossentropy, weighted_metrics=["acc"]
```

Comment:
@adocherty : flaky about changing the loss function, here, I hope it's cool to leave this as is

Reply:
Reverted change and added unsupervised bagging unit tests.